demos/CompositeUI-MVC/Divergent.ITOps.ViewModelComposition/DynamicViewModel.cs

```diff
@@ -77,9 +84,28 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
             return false;
         }
 
+        public override IEnumerable<string> GetDynamicMemberNames()
+        {
+            return base.GetDynamicMemberNames()
+                .Union(properties.Keys)
+                .Union(new[] { "RaiseEventAsync", "RaiseEvent" });
```

Comment:
yes, I wrote it in the comment. I could even be simpler:

```
foreach (var item in properties.Keys)
{
      yield return item;
}

yield return "RaiseEventAsync";
yield return "RaiseEvent";
```

dropping the `base` call, since base class has no members at all in such a scenario, never.